core/driver/sources/src/test/resources/org/virtuslab/ideprobe/dependencies/product-info.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "IntelliJ IDEA",
+  "version": "2020.2.4",
+  "buildNumber": "202.8194.7",
+  "productCode": "IC",
+  "dataDirectoryName": "IdeaIC2020.2",
+  "svgIconPath": "bin/idea.svg",
+  "launch": [
```

Comment:
I didn't know about this file. It would be nice to use it in the future to modify how we start ide probe on mac and windows and what default vm options we take. Nice find

Reply:
All credits go to @MichalBednarz, I'm just feeding on his success.